docs/index.md

```diff
@@ -4,387 +4,50 @@ jupytext:
   text_representation:
     extension: .md
     format_name: myst
-kernelspec:
-  display_name: Python 3
-  language: python
-  name: python3
 
 # override the toc-determined page navigation order
-next-page: getting_started/flyte_fundamentals
-next-page-title: Flyte Fundamentals
+next-page: getting_started/quickstart_guide
+next-page-title: Quickstart guide
 ---
 
 (getting_started_index)=
 
-# Getting Started
+# Introduction to Flyte
 
-{doc}`Example Tags <_tags/tagsindex>`
+Flyte is a workflow orchestrator that unifies machine learning, data engineering, and data analytics stacks for building robust and reliable applications. Flyte features:
+* Reproducible, repeatable workflows
+* Strongly typed interfaces
+* Structured datasets to enable easy conversion of dataframes between types, and column-level type checking
+* Easy movement of data between local and cloud storage
+* Easy tracking of data lineages
+* Built-in data and artifact visualization
 
-## Introduction to Flyte
+For a full list of feature, see the [Flyte features page](https://flyte.org/features).
 
-Flyte is a workflow orchestrator that seamlessly unifies data,
-machine learning, and analytics stacks for building robust and reliable
-applications.
+## Basic Flyte components
 
-This introduction provides a quick overview of how to get Flyte up and running
-on your local machine.
+Flyte is made up of a user plane, control plane, and data plane.
+* The **user plane** contains the elements you need to develop the code that will implement your application's directed acyclic graph (DAG). These elements are FlyteKit, the FlyteConsole, and Flytectl. Data scientists and machine learning engineers primarily work in the user plane.
+* The **control plane** is part of the Flyte backend that is configured by platform engineers or others tasked with setting up computing infrastructure. It consists of FlyteAdmin, which serves as the main Flyte API to process requests from clients in the user plane. The control plane sends workflow execution requests to the data plane for execution, and stores information such as current and past running workflows, and provides that information upon request.
+* The **data plane** is another part of the Flyte backend that contains FlytePropeller, the core engine of Flyte that executes workflows. FlytePropeller is designed as a [Kubernetes Controller](https://kubernetes.io/docs/concepts/architecture/controller/). The data plane sends status events back to the control plane so that information can be stored and surfaced to end users.
 
-````{dropdown} Want to try Flyte on the browser?
```

Comment:
I removed this link since it takes people out of the docs and puts them into an entirely different onboarding path, which seems confusing. I do think the hosted sandbox is helpful, but it seems better to link to it from the Flyte website (where it's already linked).